tests/test_creation/test_math_features.py

```diff
@@ -4,6 +4,7 @@
 from sklearn.pipeline import Pipeline
 
 from feature_engine.creation import MathFeatures
+from feature_engine.creation.custom_functions import CustomFunctions
```

Comment:
if we do keep this function, we need to test it in a separate test file. We have 1 test file per class/script